app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -678,6 +678,15 @@ const ContentForm: React.FC<IContentFormProps> = ({
                                       }
                                     />
                                   </div>
+                                  <FormikCheckbox
+                                    label="Is CBRA Unqualified"
+                                    name="isCBRAUnqualified"
+                                    checked={props.values.isCBRAUnqualified}
+                                    onChange={(e) => {
```

Comment:
You shouldn't need `onChange` or `checked` if you're using the `<FormikCheckbox/>` .  It'll take care of all that for you.

Reply:
updated. thanks